### PR TITLE
Refactor `weaviate.classes` structure

### PR DESCRIPTION
**Pull request by tsmith023**
title

### CODE REVIEW

**Review by dirkkul — APPROVED** [review]
we are moving .query out of the .config but leaving .data inside?

**Comment by tsmith023** [comment]
>we are moving .query out of the .config but leaving .data inside?

I'll make this the last split, but I won't do the same for the `.tenants` namespace

File: weaviate/collections/classes/internal.py [hunk 1]
```diff
@@ -382,13 +382,17 @@ class ReferenceAnnotation:
...
+        ...         wvc.CrossReferenceAnnotation(include_vector=True)
         ...     ]
     """
 
-    include_vector: bool = False
-    metadata: Optional[MetadataQuery] = None
-    target_collection: Optional[str] = None
+    include_vector: bool = field(default=False)
+    metadata: Optional[MetadataQuery] = field(default=None)
+    target_collection: Optional[str] = field(default=None)
```
[the remark this answers is not in the document]
Comment (tsmith023) [reply weaviate/collections/classes/internal.py:391]: Yes, when you do just `var: Optional[str] = None` the intellisense renders the type of `var` as `Literal[None]` in the `__init__` method. The `field(default=None)` undoes that quirk of `@dataclass`

[the remark this answers is not in the document]
Comment (dirkkul) [reply weaviate/collections/classes/internal.py:391]: ah nice, thanks!

**Pull request merged**